valhalla/baldr/json.h

```diff
@@ -102,7 +103,10 @@ class OstreamVisitor : public boost::static_visitor<std::ostream&>
 };
 
 inline std::ostream& operator<<(std::ostream& stream, const fp_t& fp){
-  stream << std::setprecision(fp.precision) << std::fixed << fp.value;
+  if(std::isnormal(fp.value))
```

Comment:
what about zero? should it be the without quotes for 0.0?

Reply:
i read the docs wrong